### ISSUE TITLE
currency: use kvstore as persistence layer

### ISSUE THREAD

**Issue by j4k4** [applike/gosoline#125]
replace the redis client by a configurable kvstore in the currency service in `pkg/currency/service.go:20`

the config could look like this:
```
kvstore:
  currency:
    type: chain
    application: example
    elements: [redis, ddb]
```

**Comment by applike-ss** [comment]
i would like to add inMemory here to the elements, what do you think @j4k4 ?

**Comment by ss89** [comment]
closed in #134
